x_xy/base.py

```diff
@@ -223,9 +223,14 @@ class Geometry(_Base):
     transform: Transform
     link_idx: int
 
+    color: Any = struct.field(pytree_node=False)
+    edge_color: Any = struct.field(pytree_node=False)
+
```

Comment:
Maybe add correct type hint? I believe it should be Union[str, jax.Array]

Reply:
Actually, after reading below it's probably `str | list[float]`

Reply:
Used Optional[str | tuple[float, float, float]] for now, might need to revise for other rendering backends if they want their colours in a different format?

Reply:
should be 4 floats because of rgba

Reply:
changed to support None or string or RBG or RGBA